examples/platforms/samr21/asf.h

```diff
@@ -0,0 +1,110 @@
+/**
+ * \file
+ *
+ * \brief Autogenerated API include file for the Atmel Software Framework (ASF)
+ *
+ * Copyright (c) 2012 Atmel Corporation. All rights reserved.
```

Comment:
All files outside `third_party` must carry the standard OpenThread license header.

All files that do not carry the standard OpenThread license header must be placed in `third_party`.